### ISSUE TITLE
404 blockchain not found

### ISSUE THREAD

**Issue by dshuffma-ibm** [IBM-Blockchain/ibm-blockchain-issues#75]
I'm creating this issue for @AffineR so that other people find it easier to find.

> Did you delete everyone's work when  you upgraded the fabric? 
> 
> We can't access our blockchains from April we have been using for demo purposes. How do we get them back?
> 
> We are logging on and getting "blockchain not found 404"
> 
> I want to know:
> 
> 1. Was our work of months deleted without any notice or our authority?
> 
> 2. If so, who did that? 
> 
> 3. Who in Armonk can fix this?
> 
> NOW

**Comment by dshuffma-ibm** [comment]
@AffineR
There is no ability to "upgrade" previous blockchain networks.  All active networks are kept with the same version of fabric they always had. Inactive networks however are deleted. Addressing your questions:

1. Yes, any inactive networks in the starter (free) plan are deleted after 15 days of inactivity. Which means 15 days without any block creation. We did our best to reach as many users:

  - Its in our v1.0.0 release notes:

![image](https://cloud.githubusercontent.com/assets/16085295/20671734/9332588e-b54c-11e6-9df5-c26b397acb4e.png)

  - Also our communication messages:

![image](https://cloud.githubusercontent.com/assets/16085295/20671756/a325929c-b54c-11e6-8a2a-030e1f284880.png)

These messages can be seen in the dashboard of your network or our status page. I understand only active users would have seen these notices, but it is the best we can do.

2. It was me. On behalf of the blockchain as a service team I initiated our inactive network cleanup script.

3. The network is gone, there is no backup nor any recoverable option.

To prevent this from happening.  A user must simply invoke or deploy chaincode once every 15 days.  You can also grab a new starter network at any time to continue your PoCs.

If the behavior of the starter plan is not acceptable, you may apply for a HSBN network (High Security Business Network). HSBN networks will not be pruned for block inactivity.

**Comment by AffineR** [comment]
I don't know who you are but I sincerely apologise

I have never blown my stack like that. It was a scream of frustration


We may be able to salvage some records from devopps projects

We have screen shots

It is no excuse but since 15 March our team has negotiated with 80 central
banks. Look up alliance for Financial inclusion and I'm there.

We used basic chain code for central bankers in simple demos whilst sieve
pbft and longman Schwartz algorithms took us weeks to understand!

No we don't want a hbsd at 10 K a month when we've got a now useless Z
Systems and power 8 somewhere in Texas

May I respectfully suggest you notify partner world as that's the way IBM
contacts us.

Once again apologies for losing my temper
